packages/shared/lib/core/profile-manager/api/events/transaction-progress.event.ts

```diff
@@ -10,42 +9,54 @@ import {
 } from '@core/profile-manager'
 import { isOnboardingLedgerProfile } from '@contexts/onboarding'
 import { closePopup, openPopup } from '@lib/popup'
-
 import { TransactionProgressEventPayload } from '../types'
+import { deconstructLedgerVerificationProps } from '@core/ledger/helpers'
 
 export function handleTransactionProgressEvent(accountId: string, payload: TransactionProgressEventPayload): void {
     if (get(isActiveLedgerProfile)) {
         if (get(selectedAccountId) === accountId) {
-            handleTransactionProgressInternal(payload)
+            TransactionProgressEventForLedger(payload)
```

Comment:
Should we rename this function to `handleTransactionProgressEventForLedger` since `TransactionProgressEventForLedger` doesn't quite follow the conventions / seems to add a new one?

Reply:
Whoops, I don't know where this comes from :man_shrugging: